src/common/crypto.node.test.ts

```diff
@@ -0,0 +1,21 @@
+import expect from "expect"
+
+import * as Crypto from "./crypto.js"
+
+describe("Crypto", async () => {
```

Comment:
It might be good to add an encrypt/decrypt test to cover the exchange key use case.

Reply:
Added 👍